apps/charterafrica/src/payload/blocks/VideoEmbed.js

```diff
@@ -0,0 +1,81 @@
+import blockFields from "../fields/blockFields";
+
+const VideoEmbed = {
+  slug: "videoEmbed",
+  fields: [
+    blockFields({
+      name: "videoEmbedField",
+      fields: [
+        {
+          type: "row",
+          fields: [
+            {
+              name: "embedType",
+              type: "radio",
+              defaultValue: "url",
```

Comment:
Which services do you know only share url? I just want to be sure if this is something we should be supporting now or in the future.

Reply:
I saw from wordpress pasting a youtube url works for them. Apparently they use [oEmbed Protocol](https://oembed.com/)